app/views/reviews/_review_form.html.erb

```diff
@@ -1,11 +1,11 @@
 <h1>Write Review</h1>
```

Comment:
When this renders from the Edit review page, it will say "Edit Review" then "Write Review" underneath, which might be confusing.